### PR TITLE
:nokotlin:

### PR DESCRIPTION
**Pull request by Ueaj-Kerman**
:tiny_potato:

### CODE REVIEW

**Comment by dblsaiko** [comment]
Code size increases by more than 100% but no functionality added

File: src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+ * Could not load the following classes:
+ *  io.github.alloffabric.beeproductive.api.HoneyFlavor
+ *  kotlin.Metadata
+ *  kotlin.jvm.internal.Intrinsics
+ *  net.minecraft.item.Item
+ *  net.minecraft.item.ItemConvertible
+ *  net.minecraft.item.ItemStack
+ *  net.minecraft.item.Items
+ *  org.jetbrains.annotations.NotNull
+ */
```
Comment (i509VCB) [inline src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java:13]: You were lazy enough to leave in the decompiler header

File: src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+	private static final HoneyFlavor REDSTONE_FLAVOR;
+	private static final HoneyFlavor GLOWSTONE_FLAVOR;
+	private static final HoneyFlavor COAL_FLAVOR;
+	private static final HoneyFlavor QUARTZ_FLAVOR;
+	private static final HoneyFlavor XP_FLAVOR;
+	private static final HoneyFlavor OBSIDIAN_FLAVOR;
+	private static final HoneyFlavor BLAZE_FLAVOR;
+	public static final Vanilla INSTANCE;
+
+	public final HoneyFlavor getDIAMOND_FLAVOR() {
```
Comment (i509VCB) [inline src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java:39]: make the field public

File: src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+		return DIAMOND_FLAVOR;
+	}
+
+	public final HoneyFlavor getEMERALD_FLAVOR() {
```
Comment (i509VCB) [inline src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java:43]: make the field public

File: src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+		return EMERALD_FLAVOR;
+	}
+
+	public final HoneyFlavor getIRON_FLAVOR() {
```
Comment (i509VCB) [inline src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java:47]: make the field public

File: src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+		return IRON_FLAVOR;
+	}
+
+	public final HoneyFlavor getGOLD_FLAVOR() {
```
Comment (i509VCB) [inline src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java:51]: make the field public

File: src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+		return GOLD_FLAVOR;
+	}
+
+	public final HoneyFlavor getLAPIS_FLAVOR() {
```
Comment (i509VCB) [inline src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java:55]: make the field public

File: src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+		return LAPIS_FLAVOR;
+	}
+
+	public final HoneyFlavor getREDSTONE_FLAVOR() {
```
Comment (i509VCB) [inline src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java:59]: make the field public

File: src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+		return REDSTONE_FLAVOR;
+	}
+
+	public final HoneyFlavor getGLOWSTONE_FLAVOR() {
```
Comment (i509VCB) [inline src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java:63]: make the field public

File: src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+		return GLOWSTONE_FLAVOR;
+	}
+
+	public final HoneyFlavor getCOAL_FLAVOR() {
```
Comment (i509VCB) [inline src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java:67]: make the field public

File: src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+		return COAL_FLAVOR;
+	}
+
+	public final HoneyFlavor getQUARTZ_FLAVOR() {
```
Comment (i509VCB) [inline src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java:71]: make the field public

File: src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+		return QUARTZ_FLAVOR;
+	}
+
+	public final HoneyFlavor getXP_FLAVOR() {
```
Comment (i509VCB) [inline src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java:75]: make the field public

File: src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+		return XP_FLAVOR;
+	}
+
+	public final HoneyFlavor getOBSIDIAN_FLAVOR() {
```
Comment (i509VCB) [inline src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java:79]: make the field public

File: src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java [hunk 13]
```diff
@@ -0,0 +1,117 @@
...
+		return OBSIDIAN_FLAVOR;
+	}
+
+	public final HoneyFlavor getBLAZE_FLAVOR() {
```
Comment (i509VCB) [inline src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java:83]: make the field public

File: src/main/java/com/martmists/beeresourceful/nectars/compat/TechReborn.java [hunk 14]
```diff
@@ -0,0 +1,181 @@
...
+	private static final HoneyFlavor NICKEL_FLAVOR;
+	private static final HoneyFlavor PLATINUM_FLAVOR;
+	private static final HoneyFlavor SILVER_FLAVOR;
+	private static final HoneyFlavor STEEL_FLAVOR;
+	private static final HoneyFlavor TITANIUM_FLAVOR;
+	private static final HoneyFlavor TUNGSTEN_FLAVOR;
+	private static final HoneyFlavor ZINC_FLAVOR;
+	public static final TechReborn INSTANCE;
+
+	public final HoneyFlavor getRUBBER_FLAVOR() {
```
Comment (i509VCB) [inline src/main/java/com/martmists/beeresourceful/nectars/compat/TechReborn.java:47]: make the field public

File: src/main/java/com/martmists/beeresourceful/nectars/compat/TechReborn.java [hunk 15]
```diff
@@ -0,0 +1,181 @@
...
+		return RUBBER_FLAVOR;
+	}
+
+	public final HoneyFlavor getALUMINIUM_FLAVOR() {
```
Comment (i509VCB) [inline src/main/java/com/martmists/beeresourceful/nectars/compat/TechReborn.java:51]: make the field public

**Comment by Ueaj-Kerman** [comment]
perfectly java, as all code should be

File: src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java [hunk 16]
```diff
@@ -0,0 +1,117 @@
...
+	private static final HoneyFlavor COAL_FLAVOR;
+	private static final HoneyFlavor QUARTZ_FLAVOR;
+	private static final HoneyFlavor XP_FLAVOR;
+	private static final HoneyFlavor OBSIDIAN_FLAVOR;
+	private static final HoneyFlavor BLAZE_FLAVOR;
+	public static final Vanilla INSTANCE;
+
+	public final HoneyFlavor getDIAMOND_FLAVOR() {
+		return DIAMOND_FLAVOR;
+	}
```
Comment (quat1024) [inline src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java:41]: cringe

File: src/main/java/com/martmists/beeresourceful/nectars/compat/MobZ.java [hunk 17]
```diff
@@ -0,0 +1,59 @@
+/*
+ * Decompiled with CFR 0.148.
```
Comment (quat1024) [inline src/main/java/com/martmists/beeresourceful/nectars/compat/MobZ.java:2]: cringe 2

File: src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java (hunk 16 above)
Comment (Ueaj-Kerman) [reply src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java:41]: ohyes

File: src/main/java/com/martmists/beeresourceful/nectars/compat/MobZ.java (hunk 17 above)
Comment (Ueaj-Kerman) [reply src/main/java/com/martmists/beeresourceful/nectars/compat/MobZ.java:2]: ohhhhhhhhyes

**Review by i509VCB — CHANGES_REQUESTED** [review]

File: src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java (hunk 1 above)
Comment (Ueaj-Kerman) [reply src/main/java/com/martmists/beeresourceful/nectars/Vanilla.java:13]: e

File: src/main/java/com/martmists/beeresourceful/nectars/compat/TechReborn.java (hunk 15 above)
Comment (natanfudge) [reply src/main/java/com/martmists/beeresourceful/nectars/compat/TechReborn.java:51]: Make the field public

**Pull request closed without merging**